.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn lint-staged
```

Comment:
Before committing, the repo will automatically run the lint-staged commands, which runs linting and formatting on all staged files.